lib/core/src/chain/bitcoin/esplora.rs

```diff
@@ -48,11 +48,20 @@ impl EsploraBitcoinChainService {
                 anyhow::bail!("Cannot start Bitcoin Esplora chain service without an Esplora url")
             }
         };
-        let client = Builder::new(esplora_url)
-            .timeout(3)
-            .max_retries(10)
-            .build_async()?;
+        let mut builder = Builder::new(esplora_url).timeout(3).max_retries(10);
 
+        if esplora_url == BREEZ_LIQUID_ESPLORA_URL {
```

Comment:
Shouldn't the change be only for liquid esplora? Currently it seems that we change bitcoin esplora.